content/millennium/r4/base/individuals/patient.md

```diff
@@ -11,23 +11,33 @@ title: Patient | R4 API
 
 The Patient Resource provides general demographic information about a person receiving health care services from a specific organization. Common demographic fields include patient id, patient name, gender, date of birth, address, phone, primary language and marital status. Additional concepts returned as extensions and not part of the base resource include time of birth, preferred contact, race, ethnicity and birth sex. Cerner Millennium is a patient centric application: thus, many of the other resources will include the patient id in their queries. A person receiving care from multiple organizations may have data available in multiple patient resources in multiple FHIR servers.
 
+* The following [HL7® FHIR® US Core Implementation Guide STU 4.0.0](https://hl7.org/fhir/us/core/STU4/){:target="_blank"} Profiles are supported by this resource:
+
+  * [US Core Patient Profile](http://hl7.org/fhir/us/core/STU4/StructureDefinition-us-core-patient.html){:target="_blank"}
+
 The following fields are returned if valued:
 
 * [Patient id](https://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
 * [Patient Identifier](https://hl7.org/fhir/R4/patient-definitions.html#Patient.identifier){:target="_blank"}
+  * [Usual](https://hl7.org/fhir/R4/codesystem-identifier-use.html#identifier-use-usual){:target="_blank"}
```

Comment:
Usual isn't an identifier field. Not sure the best way to represent this. Perhaps an implementation note in the read/search sections to indicate that the identifier.use will be 'usual'?